common/beerocks/btl/include/btl/btl.h

```diff
@@ -60,12 +62,19 @@ class transport_socket_thread : public socket_thread {
     int poll_timeout_ms = 500;
 
 #ifdef UDS_BUS
+    bool skip_filtered_message_type(ieee1905_1::eMessageType msg_type) override;
+
     Socket *bus = nullptr;
     std::unique_ptr<SocketServer> bus_server_socket;
 #else
     std::shared_ptr<mapf::LocalBusInterface> bus = nullptr;
     std::shared_ptr<mapf::Poller> poller         = nullptr;
 #endif
+
+    // This member supposes to be defined inside "#ifdef UDS_BUS" block, but for some reason, when
+    // defined there, there is a crash when trying to insert more than one element to the container.
+    // For now, define it here where it doesn't cause a crash.
+    std::unordered_set<ieee1905_1::eMessageType> m_subscribed_messages;
```

Comment:
Not acceptable. If you have a bug you solve it, not work around it to get the change merged.

Reply:
Vitaly fixed it.

Reply:
You marked the PR "ready for merge". If you push a commit like that and plan to fix it, it is perfectly fine, but you need to mention that in the commit message that this commit should be dropped before merge, like @rmelotte does often.